proto/viam/app/data/v1/data.proto

```diff
@@ -234,6 +234,7 @@ message GetLatestTabularDataRequest {
   string part_id = 1;
   string resource_name = 2;
   string method_name = 3;
+  string resource_type = 4;
```

Comment:
Is there a reason that this is after `method_name` instead of before?

Reply:
Yes - this is to keep `method_name` with the field number 3 which it's already been defined with. For protobuf the number each field is assigned to can't really be [changed](https://protobuf.dev/programming-guides/proto3/#assigning) once the message type is in use, so if I were to have `resource_type` "before" `method_name` here, it would require us to do something like this:

```
string resource_subtype = 4;
string method_name = 5;

reserved 3;
```

Reply:
Ooh, in my head it wasn't really in use yet. But I guess that's not true! 😬